centrifuge-js/src/modules/pools.ts

```diff
@@ -445,13 +445,19 @@ export function getPoolsModule(inst: CentrifugeBase) {
             )
           )
         )
-        if (options?.propose) {
+        if (options?.createType === 'propose') {
           const proposalSubmittable = api.tx.utility.batchAll([
             api.tx.democracy.notePreimage(submittable.method.toHex()),
             api.tx.democracy.propose(submittable.method.hash, api.consts.democracy.minimumDeposit),
           ])
           return inst.wrapSignAndSend(api, proposalSubmittable, options)
         }
+        if (options?.createType === 'notePreimage') {
+          const preimageSubmittable = api.tx.utility.batchAll([
+            api.tx.democracy.notePreimage(submittable.method.toHex()),
+          ])
```

Comment:
The batch tx is not needed in this case

```suggestion
          const preimageSubmittable = api.tx.democracy.notePreimage(submittable.method.toHex())
```